includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -331,6 +331,7 @@ public function setup_wizard_footer() {
 			<?php elseif ( 'recommended' === $this->step || 'activate' === $this->step ) : ?>
 				<a class="wc-setup-footer-links" href="<?php echo esc_url( $this->get_next_step_link() ); ?>"><?php esc_html_e( 'Skip this step', 'woocommerce' ); ?></a>
 			<?php endif; ?>
+			<?php do_action( 'wc_setup_footer' ); ?>
```

Comment:
This creates a new action yah? If so, better to name it `wc_setup_obw_footer` or something less generic.

Reply:
It does; I was trying to follow the existing patterns in the setup wizard (thought this looked pretty similar to existing naming conventions, e.g. `wp_print_scripts( 'wc-setup' );`).

I think something like `wc_obw_footer` is very clear, but the acronym `obw` doesn't seem to be used very much here.  I suppose we could rename this to `wc_admin_setup_footer` if you think there will be conflicts with this action.